modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -189,7 +189,7 @@ const get = (dom: DOMUtils) => {
     ]
   };
 
-  Tools.each('p h1 h2 h3 h4 h5 h6 div address pre div dt dd samp'.split(/\s/), (name) => {
+  Tools.each('p h1 h2 h3 h4 h5 h6 div address pre dt dd samp'.split(/\s/), (name) => {
```

Comment:
I think we discussed this, but I've forgotten the reasoning sorry? Also, should this be mentioned in the changelog?

Reply:
It's just removing the duplicate `div` from the list. There won't be any functionality change from removing the duplicate.